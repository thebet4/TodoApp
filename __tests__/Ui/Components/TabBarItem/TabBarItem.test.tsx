import renderer from 'react-test-renderer';

const TabBarItem = require('../../../../src/Ui/Components/TabBarItem').default;

describe('<TabBarItem />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(TabBarItem).toJSON()
    expect(tree).toMatchSnapshot()
  });
});
