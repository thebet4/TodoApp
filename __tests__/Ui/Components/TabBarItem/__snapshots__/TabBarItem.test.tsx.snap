// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabBarItem /> renders correctly 1`] = `null`;
